src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -523,10 +533,20 @@ private function addSerializerSection(ArrayNodeDefinition $rootNode)
     private function addPropertyAccessSection(ArrayNodeDefinition $rootNode)
     {
         $rootNode
+            ->beforeNormalization()
+                ->always(function ($v) {
+                    if (!isset($v['property_access'])) {
+                        $v['property_access']['enabled'] = true;
+                        @trigger_error('You must explicitly define wheter to enable "framework.property_access" or not. It will be disabled by default in 4.0', E_USER_DEPRECATED);
```

Comment:
As discussed https://github.com/symfony/symfony/issues/13703, it is confusing to have some components enabled by default and some others disabled by default.
BTW this deprecation can be easily fixed in the symfony standard edition.